.circleci/config.yml

```diff
@@ -189,12 +147,12 @@ jobs:
           at: /tmp
       - checkout
       - restore_cache:
-          key: conda-py37-v1-{{ checksum "setup.cfg" }}
+          key: conda-3.7-v1-{{ checksum "setup.cfg" }}
       - run:
           name: Merge coverage files
           command: |
             apt-get install -yqq curl
-            source activate phys2bids_py37  # depends on makeenv37
+            source activate 3.7
             cd /tmp/src/coverage/
```

Comment:
My guess is that we're running codecov on a directory with the coverage of the different versions, and codecov basically sees duplicates and doesn't know which one of the files it has to use. 

I would save the coverage of the different versions into a directory of their own like this:

`- src/coverage/<< parameters.python-version >>/.coverage.integration<< parameters.python-version >>`

And then when it comes to running codecov, I would run it on just one of the Python versions.

I hope that's clear.

Reply:
I thought that idea was to have it merged across all invocations/versions was as a union.

BTW, iirc codecov is ok to receive coverage submission from multiple jobs for the same pr, may be it would merge them itself?

FWIW we historically also do merging across different steps in the same Travis run in DataLad and iirc I don't remember if we had tried to just submitted them all separately